src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js

```diff
@@ -25,17 +27,16 @@ describe('<IncidentDetail />', () => {
 
   it('should render correctly', () => {
     const wrapper = shallow(
-      <IncidentDetail incident={incident} stadsdeelList={stadsdeelList} />
+      <IncidentDetail incident={incident} stadsdeelList={stadsdeelList} priorityList={priorityList} />
     );
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render correctly without extra properties', () => {
     delete incident.extra_properties;
     const wrapper = shallow(
-      <IncidentDetail incident={incident} stadsdeelList={stadsdeelList} />
+      <IncidentDetail incident={incident} stadsdeelList={stadsdeelList} priorityList={priorityList} />
     );
-    expect(wrapper.find('tr').length).toEqual(14);
```

Comment:
No replacement for this check?

Reply:
nee deze check is niet nodig want het aantal regels in de detail wordt (beter) bijgehouden in de snapshot. toch?

Reply:
I like an explicit check, as missing it in the (large) snapshot code is not so hard when updating the same or another snapshot